hpx/runtime/serialization/detail/polymorphic_intrusive_factory.hpp

```diff
@@ -157,12 +157,12 @@ namespace hpx { namespace serialization { namespace detail
 
 #define HPX_SERIALIZATION_POLYMORPHIC_TEMPLATE(Class)                         \
   HPX_SERIALIZATION_POLYMORPHIC_WITH_NAME(                                    \
-      Class, hpx::util::type_id<Class>::typeid_.type_id();)                   \
+      Class, hpx::debug::type_id<Class>::typeid_.type_id();)                   \
```

Comment:
Should this file also include the header declaring the `debug` functionality? (same is true for all references to `debug` below)

Reply:
good point there's a very old debug PR that needs to be looked at. I will get to it ...